codebase/superdarn/src.idl/lib/main.1.25/radar.pro

```diff
@@ -797,7 +797,7 @@ function RadarPos,center,bcrd,rcrd,site,frang,rsep,rxrise,$
 
 
                 
-  re=6356.779D
+  re=!CONST.R_earth/1000
```

Comment:
Won't this be different from the Earth radius defined in `rmath.h` for the C code?